src/components/shared/Buttons.js

```diff
@@ -66,7 +70,7 @@ export function ButtonSide({ action, learnMore = false }) {
     <div>
       <SideButton onClick={action}>
         <Arrow />
-        {learnMore && <LearnMessage>Learn More</LearnMessage>}
+        {learnMore && <LearnMessage>{message}</LearnMessage>}
```

Comment:
can't we `text` or caption as prop and set default to `undefined` and then render

```jsx
{caption && <Caption>{caption}</Caption>}
```

Let me know if you get the idea. We could also wrap in flex <Colum> or grid so better layout

Reply:
I don't think we need flex or grid here, but imho this should be a single prop @nitro-marky. Is there a particular reason you think separating it out as two is better?

Reply:
I felt it was clearer having the two props, 1. do I want to show it, 2. What should it say. Just separating concerns really. But I'm happy to go with caption if people prefer it.